### PR TITLE
Support Chaincoin

### PR DESCRIPTION
**Pull request by fellowserf**

### CODE REVIEW

File: NBitcoin.TestFramework/WellknownNodeDownloadData.cs [hunk 1]
```diff
@@ -793,9 +793,38 @@ public class KotoNodeDownloadData
 			};
 		}
 
+        public class ChaincoinNodeDownloadData
```
Comment (knocte) [inline NBitcoin.TestFramework/WellknownNodeDownloadData.cs]: this file has spaces instead of tabs for indentation

**Comment by NicolasDorier** [comment]
the altcoin tests are passing ?

**Comment by fellowserf** [comment]
8/10.  Both CanSyncWithPoW and CanSyncWithoutPoW are failing atm.  Will have another look.

**Comment by fellowserf** [comment]
Thanks Nicolas.  All unit tests are passing now.

**Comment by NicolasDorier** [comment]
rebase please.

**Comment by fellowserf** [comment]
All up to date.

**Pull request merged**